### PR TITLE
[MRG] [ref#116] cleaned up comments

### PR DESCRIPTION
**Pull request by the872**
as per [#116](https://github.com/pavanramkumar/pyglmnet/issues/116)

### CODE REVIEW

**Comment by jasmainak** [comment]
LGTM but can you fix the pep8 errors? thanks!

**Comment by jasmainak** [comment]
the more red lines you have in a PR, the better :)

**Comment by the872** [comment]
@jasmainak hows it look now?

File: pyglmnet/pyglmnet.py [hunk 1]
```diff
@@ -211,10 +201,6 @@ def _logL(self, beta0, beta, X, y):
         elif self.distr == 'normal':
             logL = -0.5 * np.sum((y - l)**2)
         elif self.distr == 'binomial':
-            # analytical formula
-            # logL = np.sum(y*np.log(l) + (1-y)*np.log(1-l))
-
-            # but this prevents underflow
```
Comment (jasmainak) [inline pyglmnet/pyglmnet.py]: these 4 lines look helpful. Let's keep it for now maybe?

File: pyglmnet/pyglmnet.py [hunk 2]
```diff
@@ -317,83 +294,58 @@ def fit(self, X, y):
         self : instance of GLM
             The fitted model.
         """
-        # Implements batch gradient descent (i.e. vanilla gradient descent by
-        # computing gradient over entire training set)
```
Comment (jasmainak) [inline pyglmnet/pyglmnet.py]: same here. Let's keep this comment.

File: pyglmnet/pyglmnet.py [hunk 3]
```diff
@@ -317,83 +294,58 @@ def fit(self, X, y):
...
             y[np.arange(X.shape[0]), y_bk] = 1
         else:
-            # else convert to column vector
             if y.ndim == 1:
                 y = y[:, np.newaxis]
 
-        # number of predictions
         n_classes = y.shape[1] if self.distr == 'multinomial' else 1
-
-        # Initialize parameters
```
Comment (jasmainak) [inline pyglmnet/pyglmnet.py]: this too we can keep

File: pyglmnet/pyglmnet.py [hunk 4]
```diff
@@ -317,83 +294,58 @@ def fit(self, X, y):
...
-
         fit_params = list()
 
-        # Outer loop with descending lambda
         logger.info('Looping through the regularization path')
         for l, rl in enumerate(self.reg_lambda):
             fit_params.append({'beta0': beta0_hat, 'beta': beta_hat})
             logger.info('Lambda: %6.4f' % rl)
 
-            # Warm initialize parameters
```
Comment (jasmainak) [inline pyglmnet/pyglmnet.py]: let's keep this in too

**Comment by jasmainak** [comment]
@the872 looks like you have removed all the comments! let's keep a few in -- especially the helpful ones and then you have my +1 for merge

File: pyglmnet/pyglmnet.py [hunk 5]
```diff
@@ -284,16 +264,13 @@ def _grad_L2loss(self, beta0, beta, reg_lambda, X, y):
...
-            # + reg_lambda*alpha*np.sign(beta)
 
         elif self.distr == 'binomial':
             grad_beta0 = np.sum(s - y)
             grad_beta = np.transpose(np.dot(np.transpose(s - y), X)) \
                 + reg_lambda * (1 - alpha) * beta
-            # + reg_lambda*alpha*np.sign(beta)
 
         elif self.distr == 'multinomial':
-            # this assumes that y is already as a one-hot encoding
```
Comment (jasmainak) [inline pyglmnet/pyglmnet.py]: this too looks helpful, no?

**Comment by the872** [comment]
@jasmainak sorry about that, I wasn't sure what you meant by the more red lines the better. But anyway, I fixed it up!

**Comment by jasmainak** [comment]
Haha okay :) Thanks for taking care of this @the872

**Pull request merged**